src/rustdoc.md

```diff
@@ -36,30 +36,174 @@ does is call the `main()` that's in this crate's `lib.rs`, though.)
 
 ## Cheat sheet
 
-* Use `./x.py build` to make a usable
-  rustdoc you can run on other projects.
+* Copy config.toml.example (in the root of your Rust checkout to config.toml,
+  add add the lines below. This will speed up build times dramatically by not
+  building rustc locally. And skipping minifcation will make debugging JS
+  easier.
+    download-rustc = true
+    download-ci-llvm = true
+    skip-rebuild = true
+    docs-minification = false
```

Comment:
For one, this should use inline code with `` ```toml ``. But this is handled by using `./x.py setup tools`, so beginners should already be setup. I don't think we really need this here?

Reply:
It would be nice to recommend `x.py setup tools` here; but yeah, I agree we shouldn't recommend custom configs. @jsha could you add `docs-minification = false` to `src/bootstrap/defaults/tools.toml`? I don't think `skip-rebuild` is a good option to set though, that just runs with completely the wrong version (and shouldn't be necessary with `download-ci-llvm` anyway).